b2b-sample/guardio-insurance-sample-application/util/apiDecode/dashboard/decodeMe.js

```diff
@@ -21,7 +21,12 @@ import decodeUser from "../../util/apiUtil/decodeUser";
 
 export default async function decodeMe(session) {
     try {
-        const meData = await callMe(session);
+        var meData;
+        if(session.user.id==undefined || session.user.userName==undefined || session.user.name==undefined 
```

Comment:
formatting issue . space before and after `==`. Check other places as well.

Reply:
Fixed with https://github.com/Achintha444/samples-is/commit/c56439215193067034ac953d02cb38272b2be8d4#diff-41837f5472f12624dbae0a8457f15e50adb73bf6a7f5f2aac6228a4616a1d56f